src/Planners/include/BipedalLocomotion/Planners/UnicyclePlanner.h

```diff
@@ -0,0 +1,164 @@
+/**
+ * @file UnicyclePlanner.h
+ * @authors Diego Ferigo, Stefano Dafarra
+ * @copyright 2021 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_PLANNERS_UNICYCLE_PLANNER_H
+#define BIPEDAL_LOCOMOTION_PLANNERS_UNICYCLE_PLANNER_H
+
+#include "BipedalLocomotion/Contacts/ContactList.h"
+#include "BipedalLocomotion/ParametersHandler/IParametersHandler.h"
+#include "BipedalLocomotion/System/Advanceable.h"
+
+#include <Eigen/Core>
+#include <Eigen/Dense>
+
+#include <memory>
+
```

Comment:
```suggestion
#include <optional>

```

From the CI failure

Reply:
Done in https://github.com/ami-iit/bipedal-locomotion-framework/compare/a8fad53b2e0a9d46811479f2d51659105cc91923..f0209f295c83ad3a5f1a0836e06a9cf1b616f1b2